projects/knora-ui/src/lib/viewer/values/date-value/date-value.component.html

```diff
@@ -1,19 +1,24 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-      <kui-date-input #dateInput [formControlName]="'dateValue'" class="value" placeholder="Date value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-date-input>
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-        </textarea>
-    </mat-form-field>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <div *ngIf="valueFormControl.value.end; else date">
+        <p class="rm-value date-start">Period Start: {{valueFormControl.value.start | knoraDate}}</p>
```

Comment:
At some point we should start thinking about localization so we can support other languages than English in the template.

Reply:
I concur! Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-207

Reply:
Ok, thanks for adding the DSP!